abstract_commands.xml

```diff
@@ -5,8 +5,26 @@
         is accessible, then \Fsize matching the register's actual size must be
         supported.
 
+        \begin{steps}{Perform the following sequence of operations:}
```

Comment:
I would like them to always be supported, unless it's expensive.
(If they are always implemented, then perhaps it makes sense to make \Fhalt in \Rdmcontrol read-only.)